composer.json

```diff
@@ -26,13 +26,16 @@
         "colinmollenhour/magento-redis-session": "^3.2.0",
         "cweagans/composer-patches": "^1.7",
         "magento-hackathon/magento-composer-installer": "^3.1 || ^2.1 || ^4.0",
+        "mklkj/tinymce-i18n": "^23.11",
```

Comment:
```suggestion
        "mklkj/tinymce-i18n": "^23",
```

Reply:
Why allow older versions? But bump it to 23.12?